admin/installer/cloudformation.py

```diff
@@ -0,0 +1,277 @@
+"""
+Troposphere script to generate an AWS CloudFormation JSON template.
+
+Sample usage:
+python cloudformation.py > /tmp/flocker-cluster.cloudformation.json
```

Comment:
We should move this to the documentation section.
And we should probably make a proper stub script for it `./admin/make_cloudformation` which imports functions from `./admin/cloudformation`.